mon-pix/app/components/module/_details.scss

```
@@ -85,15 +85,16 @@
   &__title {
     @extend %pix-body-xs;
 
+    display:flex;
```

ah je pensais que le lint aurait throw ici 
```suggestion
    display: flex;
```